modules/generator/processor/util/util.go

```diff
@@ -1,22 +1,23 @@
 package util
 
 import (
+	semconv "go.opentelemetry.io/otel/semconv/v1.25.0"
```

Comment:
This is changing from the `/collector` package, but this update resolves to the same string value as before.